src/QrPayment.php

```diff
@@ -556,6 +542,20 @@ public function getDueDate(): DateTimeInterface
         return $this->dueDate;
     }
 
+    /**
+     * @param string|null $binaryPath
+     *
+     * @return XzBinaryLocatorInterface
+     */
+    private function getXzBinaryLocatorByOs(?string $binaryPath = null)
```

Comment:
Please add a return type
```suggestion
    private function getXzBinaryLocatorByOs(?string $binaryPath = null): XzBinaryLocatorInterface
```